File: 70.cpp
class Solution {
public:
    int climbStairs(int n) {
        if (n==1)
            return 1;
        int a=1;
        int b=2;
        for (int i=3;i<=n;i++) {
            int t=b;
            b=a+b;
            a=t;
        }
        return b;
    }
};

File: 70.py
class Solution:
    def climbStairs(self, n: int) -> int:
        a=1
        b=2
        if n==1:
            return 1
        if n==2:
            return 2
        for i in range(3,n+1):
            a,b=b,a+b
        return b
        
